project.clj

```diff
@@ -7,10 +7,10 @@
 
   :pedantic? :abort
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [org.clojure/java.jdbc "0.6.2-alpha3"]
+                 [org.clojure/java.jdbc "0.7.5"]
                  [org.clojure/test.check "0.9.0"]
-                 [org.postgresql/postgresql "42.1.4"]
-                 [migratus "0.8.30"]
+                 [org.postgresql/postgresql "42.2.0"]
```

Comment:
We need to make sure we update the version of postgresql in clj-parent when this gets pushed there too.  https://github.com/puppetlabs/clj-parent/blob/master/project.clj#L89

Reply:
this update just removes some warnings, not strictly necessary